### PR TITLE
LPS-103684 Create an API to assign users to account roles

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-103684](https://issues.liferay.com/browse/LPS-103684).<br><br>Hey Pei-Jung, can you please look over this to see if I missed anything? Please let me know if you'd like to discuss my goal and approach<br><br>/cc @pei-jung @alee8888 @ChrisKian @dejuknow

### CODE REVIEW

**Comment by drewbrokke** [comment]
Once the code is approved, I'd like to send to Mike Han for review as well.

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java [hunk 1]
```diff
@@ -51,6 +53,34 @@ public AccountRole addAccountRole(
...
 
+	@Override
+	public void addAccountRoleUser(
+			long accountEntryId, long roleId, long userId)
+		throws PortalException {
+
+		AccountEntry accountEntry = _accountEntryLocalService.getAccountEntry(
+			accountEntryId);
+
+		_userGroupRoleLocalService.addUserGroupRoles(
```
Comment (pei-jung) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java]: Is account membership required in order to be assigned an account role? Looks like there's no check for account membership when assigning account roles.

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java [hunk 2]
```diff
@@ -51,6 +53,34 @@ public AccountRole addAccountRole(
...
+			userId, accountEntry.getAccountEntryGroupId(), new long[] {roleId});
+	}
+
+	@Override
+	public List<AccountRole> getAccountRoles(long accountEntryId, long userId)
```
Comment (pei-jung) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java]: i think it's more clear if this renames to `getUserAccountRoles`

Comment (drewbrokke) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java]: Haha, that's what I originally called it. I'll float it up to Brian to confirm the name.

File: modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java (hunk 1 above)
Comment (drewbrokke) [reply modules/apps/account/account-service/src/main/java/com/liferay/account/internal/role/AccountRoleManagerImpl.java]: Membership to an account is not required.  There is a use case where we assign a B2 user an account role. In that case they would not be an account member.

**Pull request closed without merging**